lib/oli_web/live/certificates/certificate_settings_live.ex

```diff
@@ -1,65 +1,279 @@
-defmodule OliWeb.Certificates.CertificateSettingsLive do
+defmodule OliWeb.Certificates.CertificatesSettingsLive do
   use OliWeb, :live_view
 
-  alias OliWeb.Sections.Mount
-  alias OliWeb.Common.Breadcrumb
-  alias Oli.Publishing.DeliveryResolver
   alias Oli.Delivery.Certificates
+  alias Oli.Delivery.Sections
+  alias Oli.Delivery.Sections.Section
+  alias Oli.Repo.Paging
+  alias Oli.Repo.Sorting
+  alias Oli.Publishing.DeliveryResolver
+  alias OliWeb.Certificates.CertificatesIssuedTableModel
+  alias OliWeb.Certificates.Components.CertificatesIssuedTab
+  alias OliWeb.Certificates.Components.ThresholdsTab
+  alias OliWeb.Common.Breadcrumb
+  alias OliWeb.Common.Params
+  alias OliWeb.Sections.Mount
+
+  on_mount OliWeb.LiveSessionPlugs.SetCtx
+  on_mount OliWeb.LiveSessionPlugs.SetRouteName
+  on_mount OliWeb.LiveSessionPlugs.SetUri
+
+  def mount(%{"product_id" => product_slug} = _params, session, socket) do
+    socket = assigns_for(socket, :page)
 
-  def mount(%{"product_id" => product_slug}, session, socket) do
     case Mount.for(product_slug, session) do
       {:error, e} ->
         Mount.handle_error(socket, {:error, e})
 
       {_, _, product} ->
-        {:ok,
-         assign(socket,
-           title: "Manage Certificate Settings",
-           product: product,
-           certificate: Certificates.get_certificate_by(%{section_id: product.id}),
-           breadcrumbs: breadcrumbs(product_slug),
-           graded_pages: product_graded_pages(product_slug)
-         )}
+        certificate = Certificates.get_certificate_by(%{section_id: product.id})
```

Comment:
I have a doubt, is it possible for a section to have multiple products and for each product want to be able to grant certificates? If that's the case I think this would fail